core/src/test/java/org/infinispan/container/versioning/AbstractClusteredWriteSkewTest.java

```diff
@@ -301,19 +301,22 @@ private void doTestWriteSkewWithPassivation(boolean executeOnPrimaryOwner, Opera
       }
 
       DataContainer dataContainer = TestingUtil.extractComponent(primaryOwner, DataContainer.class);
-      MagicKey[] keys = new MagicKey[MAX_ENTRIES];
+      int retryCount = 5;
       int insertCount = 10;
-      for (int i = 0; i < insertCount; ++i) {
-         for (int j = 0; j < MAX_ENTRIES; ++j) {
-            MagicKey tempKey = keys[j];
-            if (tempKey == null) {
-               tempKey = new MagicKey("other-key-" + j, primaryOwner);
-               keys[j] = tempKey;
+      for (int retry = 0; retry < 5; ++retry) {
```

Comment:
What we had before was a method we called `peek` that would return the value from the container without updating the underlying eviction structures.  This is nice to have since we may end up reading from the container more than once per command and we don't want that to increase the number of hits for a given key.

So if getIfPresent doesn't update internal eviction structure, we could definitely use that!